SuiteCareers/SuiteCareers/Views/Home/Index.cshtml

```diff
@@ -7,3 +7,52 @@
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
 </div>
 
+<div id="snapshot-bar">                                     <!--The container for the whole dang thing-->
+
+    <div class="custom-date-range">                         <!--Container for the date search fields-->
+
+        <span>Filter by Date:</span>
+        <label for="from-date"></label>                     <!--What will be populated in the from-date field initially-->
+        <input type="date" id="from-date" name="from-date"> <!--Input field for "From Date"-->
+        <span>to</span>
+        <label for="to-date"></label>                       <!--What will be populated in the to-date field initially-->
+        <input type="date" id="to-date" name="to-date">     <!--input field for "To Date"-->
```

Comment:
Wrapped in a `<form asp-controller="Users" asp-action="filterByDate" method="post">...</form>` and include a "Filter" button

Reply:
If switching it to bootstrap makes it better than I can try. The problem is that I don't know how to do that